sklearn/tests/test_cross_validation.py

```diff
@@ -1250,3 +1254,30 @@ def test_cross_val_predict_sparse_prediction():
     preds_sparse = cval.cross_val_predict(classif, X_sparse, y_sparse, cv=10)
     preds_sparse = preds_sparse.toarray()
     assert_array_almost_equal(preds_sparse, preds)
+
+
+def test_rfe_cv():
```

Comment:
Silly me, this shouldn't be in test_cross_validation. It should be in feature_selection/tests/test_rfe.py